app/src/main/java/com/example/sharingang/utils/notification/NotificationUtils.kt

```diff
@@ -20,13 +20,15 @@ private const val NOTIFICATION_ID = 0
  * @param messageBody the message to be displayed in the notification
  * @param applicationContext the context of the application
  */
-fun NotificationManager.sendNotification(messageBody: String, applicationContext: Context) {
-    val contentIntent = Intent(applicationContext, MainActivity::class.java)
+fun NotificationManager.sendNotification(messageBody: String, deeplink: String, applicationContext: Context) {
+    val contentIntent = Intent(Intent.ACTION_VIEW, Uri.parse(deeplink))
+    contentIntent.setPackage("com.example.sharingang")
```

Comment:
In case we change the package name
```suggestion
    contentIntent.setPackage(BuildConfig.APPLICATION_ID)
```

Reply:
Oh yeah sure great catch.